tests/python/tank_test/tank_test_base.py

```diff
@@ -347,6 +348,12 @@ def setUp(self, parameters=None):
                            - 'primary_root_name': 'name' - Set the primary root name, default to 'unit_tests'.
 
 
+        """
+        self._setUp(parameters)
```

Comment:
I've refactored how these methods are invoked so we get a clear distinction between time spent on both classes during tearDown and setUp, as the do_io has a huge effect on performance.

This became necessary as I spend a couple hours investigating poor IO performance on Windows machines on Azure. Unfortunately, I wasn't able to extract more speed.